appstream.patch

```diff
@@ -0,0 +1,25 @@
+From a45c1620c5d12b9ae4bd7b0cad9aaec1abdcef92 Mon Sep 17 00:00:00 2001
+From: Justin Zobel <justin.zobel@gmail.com>
+Date: Mon, 15 Apr 2024 08:16:52 +0000
+Subject: [PATCH] Update org.kde.akregator.appdata.xml - Drop .desktop suffix
+ from Appstream ID
+
+---
+ src/data/org.kde.akregator.appdata.xml | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/data/org.kde.akregator.appdata.xml b/src/data/org.kde.akregator.appdata.xml
+index 684acce5e..52918d9b0 100644
+--- a/src/data/org.kde.akregator.appdata.xml
++++ b/src/data/org.kde.akregator.appdata.xml
+@@ -1,6 +1,6 @@
+ <?xml version="1.0" encoding="utf-8"?>
+ <component type="desktop">
+-  <id>org.kde.akregator.desktop</id>
++  <id>org.kde.akregator</id>
```

Comment:
Try adding a [launchable](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#launchable) instead of changing the id. That should give some info.

Reply:
Actually never mind, this suffers from the same issue, metainfo has a typo in launchable
it should be `org.kde.akregator` not `org.kde.akgregator.desktop`

Reply:
Thanks! https://invent.kde.org/pim/akregator/-/merge_requests/50